earth_enterprise/src/fusion/autoingest/StorageManagerAssetHandle_unittest.cpp

```diff
@@ -63,6 +64,22 @@ TEST(AssetHandleTest, Finalize) {
   ASSERT_TRUE(finalized);
 }
 
+TEST(AssetHandleTest, ConvertConstToMutable) {
```

Comment:
I'm a little confused on this function. `ConvertConstToMutable` sounds like we want to start with a const handle and create a mutable handle to the same resource. But this test looks like it's doing it the other way around: creating a const handle from a mutable handle.

Reply:
You're right, I got the name backwards.